Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu
+MAINTAINER Alex Soto <asotobueno@cloudbees.com> 
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Install wget and unzip
+RUN apt-get update --yes \
+  && apt-get install --yes curl unzip
+
+# Install Packer
+
+ENV PACKER_BIN_DIR /packer/bin
```

Comment:
I usually do in this way or in `/usr/local` @ndeloof just let me know your preference and I change because I think that in this sense everyone has its own preference.